react/week1/todolist/src/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import TodoApp from './TodoApp'
+import reportWebVitals from './reportWebVitals';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <TodoApp />
+    {/* <App /> */}
```

Comment:
You should always put the main component here which includes all the other components and that usually is `<App />`.  Since you only have one component here, this approach works, but what if you have two ? you can't put both of them here, right?